website/package.json

```diff
@@ -7,16 +7,18 @@
     "@hashicorp/mktg-global-styles": "2.1.0",
     "@hashicorp/nextjs-scripts": "16.2.0",
     "@hashicorp/react-button": "4.0.0",
-    "@hashicorp/react-docs-page": "10.9.4-alpha.18",
+    "@hashicorp/react-docs-page": "11.0.1",
     "@hashicorp/react-hashi-stack-menu": "^1.1.0",
     "@hashicorp/react-head": "1.1.6",
     "@hashicorp/react-inline-svg": "5.0.0",
     "@hashicorp/react-markdown-page": "^0.1.0",
     "@hashicorp/react-product-downloader": "4.0.2",
-    "@hashicorp/react-search": "^3.0.0",
+    "@hashicorp/react-search": "^4.2.0",
     "@hashicorp/react-section-header": "4.0.0",
     "@hashicorp/react-subnav": "7.1.0",
     "@hashicorp/react-vertical-text-block-list": "4.0.1",
+    "adm-zip": "^0.5.4",
+    "gray-matter": "^4.0.2",
```

Comment:
`gray-matter` is used to parse `nav_title` or `sidebar_title` values from `docs.mdx` files, so we can construct `nav-data` for plugin docs with human-readable titles (rather than just using `.mdx` file basenames).